token/program/src/processor.rs

```diff
@@ -757,6 +758,27 @@ impl Processor {
         Ok(())
     }
 
+    /// Processes a [GetAccountDataSize](enum.TokenInstruction.html) instruction
+    pub fn process_get_account_data_size(accounts: &[AccountInfo]) -> ProgramResult {
+        let account_info_iter = &mut accounts.iter();
+        // make sure the mint is passed in
+        let _mint_account_info = next_account_info(account_info_iter)?;
```

Comment:
Should we unpack the mint to verify it is a valid mint?

Reply:
Will do